packages/web/src/components/gcds-link/gcds-link.css

```diff
@@ -0,0 +1,30 @@
+:host a {
+  cursor: pointer;
+  color: var(--gcds-link-default);
+
+  &:not([aria-disabled='true']) {
+    text-decoration: underline;
+    text-underline-offset: var(--gcds-link-underline-offset);
+    text-decoration-color: currentColor;
+    text-decoration-thickness: var(--gcds-link-decoration-thickness);
+  }
+
+  &:hover {
+    color: var(--gcds-link-hover);
+    text-decoration-thickness: var(--gcds-link-hover-decoration-thickness);
+  }
+
+  &:focus {
+    background-color: var(--gcds-link-focus-background);
+    color: var(--gcds-link-focus-text);
+    outline: var(--gcds-link-focus-outline-width) solid var(--gcds-link-focus-background);
+    outline-offset: var(--gcds-link-focus-outline-offset);
+    border-color: var(--gcds-link-focus-border-color);
+    text-decoration: none;
+  }
+
+  /* Small link styling */
+  &.link--small {
+   font: var(--gcds-link-small-font);
+  }
+}
```

Comment:
Just realized:

Since we are using a slot in this component, we need to use the slot bug fix that Ethan worked on this week:
```
slot {
    display: inherit;
  }
  ```

Reply:
Good cach